python/init.sls

```diff
@@ -28,18 +27,17 @@ python-headers:
       - libxslt1-dev
       - ghostscript
 
-setuptools:
-  pip.installed:
-    - name: setuptools<8.0
-    - upgrade: True
-    - require:
-      - pkg: python-base-pkgs
-
 pip:
+  cmd.run:
+    - unless: test -x /usr/local/bin/pip
+    - name: |
+        cd /tmp
+        wget -c https://bootstrap.pypa.io/get-pip.py
+        python get-pip.py
+    - cwd: /tmp
+    - user: root
   pip.installed:
```

Comment:
I believe that we want to maintain the ability for Salt to upgrade pip as applicable, no?

The bootstrapping will only run once, but pip upgrades can occur upon deploys that occur post-new pypi releases.